app/src/pages/trace/AnnotationActionMenu.tsx

```diff
@@ -1,21 +1,54 @@
-import React from "react";
+import React, { useMemo } from "react";
 
 import { ActionMenu, Flex, Icon, Icons, Item, Text } from "@arizeai/components";
 
 type AnnotationActionMenuProps = {
   onDelete: () => void;
+  onEdit?: () => void;
   isDisabled?: boolean;
 };
 
 enum AnnotationAction {
   DELETE = "deleteAnnotation",
+  EDIT = "editAnnotation",
 }
 
 /**
  * A generic action menu for annotations that can be extended
  */
 export function AnnotationActionMenu(props: AnnotationActionMenuProps) {
-  const { onDelete, isDisabled = false } = props;
+  const { onDelete, isDisabled = false, onEdit } = props;
+  const actionItems = useMemo(() => {
+    const items = [
+      <Item key={AnnotationAction.DELETE}>
+        <Flex
+          direction="row"
+          gap="size-75"
+          justifyContent="start"
+          alignItems="center"
+        >
+          <Icon svg={<Icons.TrashOutline />} />
+          <Text>Delete</Text>
+        </Flex>
+      </Item>,
+    ];
+    if (onEdit) {
+      items.unshift(
+        <Item key={AnnotationAction.EDIT}>
+          <Flex
+            direction="row"
+            gap="size-75"
+            justifyContent="start"
+            alignItems="center"
+          >
+            <Icon svg={<Icons.Edit2Outline />} />
+            <Text>Edit</Text>
+          </Flex>
+        </Item>
+      );
+    }
+    return items;
+  }, [onEdit]);
```

Comment:
Can we make it so that annotations are always in edit mode for human? They already clicked on "annotate"

Reply:
yes makes sense will fix that up!